components/ActivityForm/index.js

```diff
@@ -98,14 +108,17 @@ export default function ActivityForm({ activity, onSubmit, isUpdateMode }) {
           defaultValue={activity?.area}
         ></StyledInputs>
         <label htmlFor="country">Country: </label>
-        <StyledInputs
+        <StyledSelect
           id="country"
           name="country"
-          type="text"
-          placeholder="Please choose a country.."
-          maxLength="30"
-          defaultValue={activity?.country}
-        ></StyledInputs>
+          options={countriesOptions}
+          placeholder="Please choose country.."
+          value={
+            isUpdateMode
+              ? { value: activity.country, label: activity.country }
+              : activity
+          }
```

Comment:
Unfortunately when I'm in edit mode I can't select another country. Maybe you could solve the problem by using default value instead of value:
  ```suggestion
          defaultValue={
            isUpdateMode
              ? { value: activity.country, label: activity.country }
              : activity
          }
```